docs/source/upgrade_notes.rst

```diff
@@ -56,9 +56,19 @@ Upgrade Notes
   key_value_pair permissions.  
   Further information can be found in the :doc:`RBAC documentation <rbac>`.
 
-* Additional garbage collection options are available to automatically delete old tokens.
+* Additional garbage collection options are available to automatically delete old tokens. 
+  New options include `tokens_ttl` to expire old tokens.
   See :doc:`purging old data documentation <troubleshooting/purging_old_data>` for further
   information.
+  
+ * StackStorm’s Workflow engine can now handle shutdown events more gracefully.
+   Two new configurations has been added to `st2.conf` in the `[workflow_engine]` section to have more granular control.
```

Comment:
Upgrade notes are intended to list the changes that are really required for the user to be aware of when upgrading from one st2 version to another (schema changes, renames, breaking changes, OS changes, python version, etc)

New features and enhancements like this one are usually not mentioned in detail for "Upgrade Notes".